ui/panels-plugin/src/plugins/line-chart/LineChartOptionsEditor.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { OptionsEditorProps } from '@perses-dev/plugin-system';
+import { Box } from '@mui/material';
+import { LineChartOptions } from './line-chart-model';
+
+export type LineChartOptionsEditorProps = OptionsEditorProps<LineChartOptions>;
+
+export function LineChartOptionsEditor(props: LineChartOptionsEditorProps) {
+  const { value } = props;
```

Comment:
Btw, what is value in all these? Like what is it representing? I can probably also just look this info up by looking at some models though.

Reply:
Value is just the generic `Options` type that plugin authors control. At runtime (in the app/framework code), we don't know what they used, so it's just a `JsonObject` then.